octofree/main.py

```
@@ -98,23 +101,65 @@ def should_check_x():
+    - 5-minute warnings before session ends
+    
+    Args:
+        check_interval (int): Seconds between checks (default: 60)
+    """
+    global reminder_thread_running
+    
+    while reminder_thread_running:
+        try:
+            # Check and send any due reminder notifications
+            check_and_send_notifications()
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)